hack/update-import-aliases.sh

```diff
@@ -8,20 +8,20 @@ SCRIPT_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 cd "${SCRIPT_ROOT}"
 ROOT_PATH=$(pwd)
 
-temp_path=$(mktemp -d)
-pushd "${temp_path}" >/dev/null
-cp "${ROOT_PATH}"/go.mod .
-GO111MODULE=on go get "k8s.io/kubernetes/cmd/preferredimports@v1.21.3"
```

Comment:
and here why we not install preferredimports anymore? :)

Reply:
I explained why we don't install preferredimports by `go install` in the comments below:

```
# We can't directly install preferredimports by `go install` due to the go.mod issue:
# go install k8s.io/kubernetes/cmd/preferredimports@v1.21.3: k8s.io/kubernetes@v1.21.3
#	  The go.mod file for the module providing named packages contains one or
#	  more replace directives. It must not contain directives that would cause
#	  it to be interpreted differently than if it were the main module.
```